### ISSUE TITLE
TechDraw: More intuitive location for prefs to show/hide selection handles

### ISSUE THREAD

**Issue by maxwxyz** [FreeCAD/FreeCAD#13473]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

From the FreeCAD Day 2024 Complaint Session:
**In Techdraw there are selection handles placed by default that are shown in user interface, it’s hard to find where to turn those off.**

Discussion:
In TechDraw it puts the big blobs on things, which is nice for selecting them, but it's something if you're trying to work out what your thing actually looks like.
There's loads of this big blobs and it doesn't actually look like your technical drawing (frames, iso points,...)
In tech draw there are selection handles that are placed there by default that are shown in the user interface. That are not actually in the drawing and it is very hard to find where to switch those off.

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36807 (Git)
Build type: Release
Branch: main
Hash: cc96f2718e139d670e08c4226bdea9955a102ed6
Python 3.11.8, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: German/Germany (de_DE)
Installed mods: 
  * BIM 2021.12.0
  * Curves 0.6.29
  * fasteners 0.5.16
  * OpenDark 2023.12.17
  * sheetmetal 0.4.11
```

### Subproject(s) affected?

Techdraw

### Anything else?

Related:
- https://github.com/FreeCAD/FreeCAD/issues/11635

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by WandererFan** [comment]
Is RMB context menu with toggle frames really hard to find?

**Comment by maxwxyz** [comment]
I don't think so, and there is also a toolbar button. But this was a complaint which was mentioned. Maybe it's more about the iso points view settings but I cannot tell as I was not present during that day.

**Comment by ryankembrey** [comment]
@maxwxyz This is potentially closed by auto show frames on hover?
